src/index.html

```diff
@@ -3,11 +3,640 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
+    <title>Air</title>
+    <link rel="icon" href="./images/favicon.png">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&family=Poppins:wght@600;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Dia</h1>
+  <body class="page">
+    <header class="header" id="home">
+      <div class="header__container">
+        <div class="header__top-bar top-bar">
+          <div class="header__top--bar-content">
+            <a href="#" class="header-logo">
+              <img
+              class="header__logo"
+              src="./images/logo/logo.png"
+              alt="logo"
+              >
+            </a>
+            <a class="header__menu-content" href="#menu"></a>
+          </div>
+
+          <nav class="menu__nav">
+            <ul class="menu__list">
+              <li class="menu__item">
+                <a class="menu__link" href="#about">About us</a>
+              </li>
+
+              <li class="menu__item">
+                <a class="menu__link" href="#services">Services</a>
+              </li>
+
+              <li class="menu__item">
+                <a class="menu__link" href="#testimonials">Testimonials</a>
+              </li>
+
+              <li class="menu__item">
+                <a class="menu__link" href="#contact">Contact Us</a>
+              </li>
+            </ul>
+          </nav>
+
+          <div class="header__menu">
+            <a class="header__top-title" href="#contact">Hire us</a>
+            <div class="header__menu-border"></div>
+          </div>
+        </div>
+      </div>
+
+      <div class="header__container-section">
+        <div class="header__container-content">
+          <div class="header__container-block">
+            <h1 class="header__container-tittle">Strategic Agency</h1>
+
+            <h2 class="header__container--text">
+              We believe in the power of bold ideas that can solve business challenges.
+            </h2>
+
+            <a class="header__container--button" href="#work">
+              <button class="header__container--button-text">Learn more</button>
+            </a>
+          </div>
+
+          <div class="header__bottom">
+            <div class="header__bottom-image">
+              <div class="header__bottom-container">
+                <div class="header__bottom-button">
+                  <button class="header__bottom-click">
+                    <img
+                    class="header__bottom--click-img"
+                    src="./images/slider/chevron_left.png"
+                    alt="click"
+                    >
```

Comment:
The same as above, fix it everywhere
```suggestion
                    <img
                      class="header__bottom--click-img"
                      src="./images/slider/chevron_left.png"
                      alt="click"
                    >
```